### PR TITLE
Improve the log level display

### PR DESCRIPTION
**Pull request by annando**
This improves PR #14135 by adding more loglevels.

### CODE REVIEW

File: view/global.css [hunk 1]
```diff
@@ -773,3 +773,13 @@ figure.img-allocated-height img{
...
+.loglevel-alert {
+	color: #ff0000; /* red */
+	font-weight: bold;
+  font-style: italic;
+}
+.loglevel-emergency {
+	color: #731289; /* purple */
+	font-weight: bold;
+  font-style: italic;
+}
```
Comment (MrPetovan) [inline view/global.css]: Please normalize white space to tabulations:
```suggestion
.loglevel-alert {
	color: #ff0000; /* red */
	font-weight: bold;
	font-style: italic;
}
.loglevel-emergency {
	color: #731289; /* purple */
	font-weight: bold;
	font-style: italic;
}
```

File: view/theme/frio/templates/admin/logs/view.tpl [hunk 2]
```diff
@@ -65,11 +65,14 @@
 					data-data="{{$row->data}}" data-source="{{$row->source}}">
 					<td>{{$row->date}}</td>
 					<td class="
-						{{if $row->level == "CRITICAL"}}bg-danger
-						{{elseif $row->level == "ERROR"}}bg-danger
+						{{if $row->level == "EMERGENCY"}}bg-danger
+						{{elseif $row->level == "ALERT"}}bg-danger
+						{{elseif $row->level == "CRITICAL"}}bg-warning
+						{{elseif $row->level == "ERROR"}}bg-warning
```
Comment (MrPetovan) [inline view/theme/frio/templates/admin/logs/view.tpl]: We don't really distinguish between these 4 log levels in practice, so I would set them all to `bg-danger`. On the other hand, warnings can often be ignored.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/templates/admin/logs/view.tpl (hunk 2 above)
Comment (annando) [reply view/theme/frio/templates/admin/logs/view.tpl]: This "on the other hand" confuses me. Should I change it?

Comment (MrPetovan) [reply view/theme/frio/templates/admin/logs/view.tpl]: What I'm saying is that it is counter-productive to "hide" error and critical messages behind the Warning color which doesn't necessarily prompts an admin to react to these messages. Please have only warnings use the `bg-warning` class, all the worse levels should use the `bg-danger` class.

Comment (annando) [reply view/theme/frio/templates/admin/logs/view.tpl]: Changed

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**